### PR TITLE
Add openshift-descheduler project.

### PR DESCRIPTION
**Pull request by aveshagarwal**
This PR adds openshift-descheduler project for descheduler role.

@ingvagabund @michaelgugino @sjenning @derekwaynecarr

### CODE REVIEW

File: roles/openshift_descheduler/defaults/main.yaml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 # descheduler common setup
 openshift_descheduler_state: present
-openshift_descheduler_namespace: openshift-infra
```
Comment (ingvagabund) [inline roles/openshift_descheduler/defaults/main.yaml]: imho s/openshift-infra/openshift-descheduler would make the PR smaller and easier to change the namespace in the future

Comment (aveshagarwal) [reply roles/openshift_descheduler/defaults/main.yaml]: As discussed offline, we do not want to make it configurable.

Comment (ingvagabund) [reply roles/openshift_descheduler/defaults/main.yaml]: What a shame the Ansible has no concept of constant variables then.

**Pull request merged**